src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.Gss.cs

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+using size_t = System.IntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class libgssapi
```

Comment:
The SSL one may have gotten missed during the cleanup (maybe had a parallel PR).

System.Globalization.Native/... all uses `Interop.GlobalizationNative`
System.Native/... all uses `Interop.Sys`
System.Net.Http.Native/... all uses `Interop.Http`
System.Security.Cryptography.Native/... all (except Interop.Ssl.cs) uses `Interop.Crypto`.

To me, the files being added now would be `Interop.NetSecurity`.